### ISSUE TITLE
Export SCIRun scenes from ViewScene

### ISSUE THREAD

**Issue by a-y-khan** [SCIInstitute/SCIRun#691]
Export objects from a SCIRun scene into an appropriate (3D?) file format. Not possible with the current ViewScene implementation.

**Comment by dcwhite** [comment]
@jessdtate Solicit ideas from interns for this?

**Comment by jessdtate** [comment]
Yeah, if there is a easy way to do this, I don't think anyone would say no, but I'm not sure that it is in as much demand.  There are some os native readers for stl, for example

**Comment by dcwhite** [comment]
"SCIRan"/"SCIRen" idea: split off ViewScene into a stand-alone app. Can be built alongside SCIRun.
- [ ] Examine VS dependencies. Needs API to build new app easily.
- [ ] Add save/load for Geometry objects. Use spire/ES buffer logic?
- [ ] Either include extra VS logic or have a dummy network behind the scenes that feeds VS objects.

**Comment by dcwhite** [comment]
**Triage 2026-06:** Export SCIRun scenes — saving geometry objects to 3D files (OBJ, STL, PLY) from ViewScene. The current ViewScene has no export path; geometry is only accessible through the spire/ES buffer system. A viable approach is to intercept the VBO/IBO data in `SRInterface::handleGeomObject()` and serialize it. High effort. Related to #426 (PLY format) and #1239 (OBJ format).
